backend/preprocessing/tst_data_lib.py

```diff
@@ -769,51 +769,57 @@ def _validate_tests(self):
             # Check mandatory columns
             for col_constraint in (
                 {
-                    "path": "specimen number",
+                    "paths": ("specimen number",),
                     "mandatory": True,
                 },
                 {
-                    "path": "stress ratio",
+                    "paths": ("stress ratio",),
                     "mandatory": Experiment.__get_val_at(
                         self.experiment, "general>experiment type", ""
                     )
                     == "FA",
                     "mandatory_condition": "when experiment type is 'FA'",
                 },
                 {
-                    "path": "maximum stress",
+                    "paths": ("maximum stress",),
                     "mandatory": Experiment.__get_val_at(
                         self.experiment, "general>experiment type", ""
                     )
                     == "FA",
                     "mandatory_condition": "when experiment type is 'FA'",
                 },
                 {
-                    "path": "frequency",
+                    "paths": (
+                        "frequency",
+                        "displacement controlled loading rate",
+                        "load controlled loading rate",
+                    ),
                     "mandatory": Experiment.__get_val_at(
                         self.experiment, "general>experiment type", ""
                     )
                     == "FA",
                     "mandatory_condition": "when experiment type is 'FA'",
                 },
                 {
-                    "path": "run out",
+                    "paths": ("run out",),
                     "mandatory": Experiment.__get_val_at(
                         self.experiment, "general>experiment type", ""
                     )
                     == "FA",
                     "mandatory_condition": "when experiment type is 'FA'",
                 },
             ):
-                if (
-                    col_constraint["mandatory"]
-                    and col_constraint["path"] not in self.tests.columns
-                ):
-                    self.logger.error(
-                        "missing mandatory column "
-                        f"{col_constraint.get('mandatory_condition', '')}: "
-                        f"{col_constraint['path']}"
-                    )
+                if col_constraint["mandatory"]:
+                    found = False
+                    for path in col_constraint["paths"]:
+                        if path in self.tests.columns:
+                            found = True
```

Comment:
@sbancal J'ai l'impression qu'un while ou autre chose serai mieux. du genre.
`found = any(path in self.tests.columns for path in col_constraint["paths"])`
Plutot qu'une boucle for pas débranchange

Reply:
I like your suggestion ... thnks!
implemented!